src/librustc_resolve/lib.rs

```diff
@@ -1440,6 +1451,49 @@ impl<'a> Resolver<'a> {
         }
     }
 
+    fn find_unique_symbols(&mut self) {
```

Comment:
Could you move this to `diagnostics.rs`?

Reply:
Done